be/src/runtime/collection_value.h

```diff
@@ -160,7 +160,7 @@ class ArrayIterator {
         }
         return false;
     }
-    bool seek(uint32_t n) const {
+    bool seek(int64_t n) const {
```

Comment:
If using `int64_t` here, the function `seek` should check whether `n` is negative.